src/includes/gateways/inpage/class-alma-payment-gateway-in-page.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Alma_Payment_Gateway_In_Page.
+ *
+ * @since 5.0.0
+ *
+ * @package Alma_Gateway_For_Woocommerce
+ * @subpackage Alma_Gateway_For_Woocommerce/includes/gateways/inpage
+ * @namespace Alma\Woocommerce\Gateways\InPage
+ */
+
+namespace Alma\Woocommerce\Gateways\Inpage;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use Alma\Woocommerce\Exceptions\Alma_No_Credentials_Exception;
+use Alma\Woocommerce\Gateways\Alma_Payment_Gateway;
+use Alma\Woocommerce\Helpers\Alma_Cart_Helper;
+use Alma\Woocommerce\Helpers\Alma_Constants_Helper;
+
+/**
+ * Alma_Payment_Gateway_In_Page
+ */
+class Alma_Payment_Gateway_In_Page extends Alma_Payment_Gateway {
+
+	/**
+	 * The cart helper.
+	 *
+	 * @var Alma_Cart_Helper
+	 */
+	protected $cart_helper;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param boolean $check_basics Basic checks.
+	 *
+	 * @throws Alma_No_Credentials_Exception The exception.
+	 */
+	public function __construct( $check_basics = true ) {
+		$this->id                 = $this->get_gateway_id();
+		$this->has_fields         = $this->has_fields();
+		$this->cart_helper        = new Alma_Cart_Helper();
+		$this->method_title       = __( 'Payment in instalments and deferred with Alma - 2x 3x 4x', 'alma-gateway-for-woocommerce' );
+		$this->method_description = __( 'Install Alma and boost your sales! It\'s simple and guaranteed, your cash flow is secured. 0 commitment, 0 subscription, 0 risk.', 'alma-gateway-for-woocommerce' );
+		parent::__construct( $check_basics );
+	}
+
+	/**
+	 * Get the gateway id.
+	 *
+	 * @return string
+	 */
+	public function get_gateway_id() {
+		return Alma_Constants_Helper::GATEWAY_ID_IN_PAGE;
+	}
+
+	/**
+	 * Has fields.
+	 *
+	 * @return true
+	 */
+	public function has_fields() {
+		return true;
+	}
```

Comment:
Why this function just for return true ?

Reply:
yes, it's a woocommerce function for the gateways, it can be yes or no